packages/connectors/connector-oidc/src/types.ts

```diff
@@ -30,6 +31,19 @@ export const idTokenProfileStandardClaimsGuard = z.object({
   nonce: z.string().nullish(),
 });
 
+// Extend `idTokenProfileStandardClaimsGuard` by accepting string-typed boolean claims.
+export const idTokenProfileStandardClaimsGuardAcceptingStringTypedBooleanClaims =
+  idTokenProfileStandardClaimsGuard.omit({ email_verified: true, phone_verified: true }).extend({
+    email_verified: z
+      .boolean()
+      .nullish()
```

Comment:
Should we place the nullish statement at the end?

Reply:
Updated, but the actually logic could be the same w/ or w/o the change.